eq-author/src/components/modals/QuestionnaireSelectModal/index.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+import { colors } from "constants/theme";
+import Modal from "components/modals/Modal";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+import ScrollPane from "components/ScrollPane";
+import Panel from "components/Panel";
+
+const ModalFooter = styled.div`
+  padding: 1.5em;
+  border-top: 1px solid ${colors.bordersLight};
+`;
+
+const StyledModal = styled(Modal)`
+  .Modal {
+    padding: 0;
+    width: 55em;
+  }
+`;
+
+const Container = styled.div`
+  background: white;
+`;
+
+const ModalTitle = styled.div`
+  font-weight: bold;
+  font-size: 1.2em;
+  color: ${colors.text};
+  margin-bottom: 0.75em;
+`;
+
+const ModalSubtitle = styled.div`
+  font-size: 1em;
+  color: ${colors.text};
+`;
+
+const ModalHeader = styled.div`
```

Comment:
```suggestion
const ModalHeader = styled.header`
```

Would be good I think if we as a team start using HTML5 elements, as they're more semantic and some assistive technologies hook onto them when giving aid. 

You can use `header` as both a heading for the whole page, or as a header for a section of content on the page.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header